README.md

```diff
@@ -1 +1,65 @@
+
+Because these files are available publicly, we have no record of who downloads them. Therefore, you should sign up for notifications. Specifically, if you want to receive updates if we uncover any problems, it is imperative that you either (a) [sign up for the SAMPL e-mail list](https://mailchi.mp/e36018629725/sampl9-sign-ups), or (b) sign up for notifications of changes to this GitHub repository (the Watch button, above); ideally you would do both. 
+
+Please note that some aspects of the SAMPL8 series of challenges are still ongoing, but as we are launching a new host-guest challenge that marks the beginning of the SAMPL9 series of challenges, so we have opened up this repository. 
+
+## Achnowleding and citing SAMPL
+
+If you've benefitted from our work on the SAMPL series of challenges, please be sure to acknowledge our SAMPL NIH grant in any publications/presentations. This funded host-guest experimetns, as well as our work organizing and administrating these challenges. You may acknowledge SAMPL by saying thing like, "We appreciate the National Institutes of Health for its support of the SAMPL project via R01GM124270 to David L. Mobley (UC Irvine)."
+
+We also ask you to cite the SAMPL dataset(s) you used. These are versioned on Zenodo, and the latest DOI is here: [![DOI](svg item goes here)](badge latestdoi will go here ). Click through for access to all data releases. You may cite these sets by their DOI. 
+
+Of course, we also appreciate it if you cite any overview/experimental papers relevant to the particular SAMPL challenge you participated in. 
+
+## What's here
+- Host-guest challenge files for the WP6 challenge
+- [Host-guest participation instructions](https://github.com/samplchallenges/SAMPL9/blob/master//host_guest_instructions.md) with information on the submissions format, etc. Submission templates are available in the subdirectories for individual host-guest systems. 
+
+## What's coming
+
+Experimental results for WP6. 
+
+## Changes and Data Set Versions
+
+### Releases
+
+
+
+### Changes not in a release
+
+- Add WP6 challenge details, deadline, host/guest files, submission template, and submission instructions
+
+## Challenge construction
+
+### Overview
+
+The SAMPL9 phase of challenges includes a new host-guest challenge (WP6). We are currently running our physical properties challenge with GSK including pKa and logD prediction. 
```

Comment:
I'd remove the "We are currently running..." portion since that's SAMPL8 and this is the SAMPL9 repo.